<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaintainceLoadPage</class>
 <widget class="QWidget" name="MaintainceLoadPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:black;
}
QLabel
{
	background-color:rgb(0,100,255);
	color:white;
	border-color:transparent;
    font:20px &quot;微软雅黑&quot;;
}</string>
  </property>
  <widget class="QLabel" name="TrainWeightTC1lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainWeightMP1lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainWeightM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainWeightM2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainWeightMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="TrainWeightTC2lbl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>40</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="returnBtn">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>530</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
	font: 20px, &quot;微软雅黑&quot;;
    color: white;
		border-radius:8px;
background-color: rgb(0, 128, 0);
    border-top: 2px solid white;
	border-left: 2px solid white;

}</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>520</y>
     <width>1018</width>
     <height>72</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(0, 0, 0);

	border:1px solid white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>车辆重量（t）</string>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightM2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightTC2lbl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightTC1lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="EmptyTrainWeightMP1lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>120</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>空车重量（t）</string>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightM2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightTC2lbl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightTC1lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PkgWeightMP1lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>载客量（kg）</string>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightTC2lbl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightMP1lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>载客量（％）</string>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightM2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PperWeightTC1lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>280</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumMP2lbl">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumTC2lbl">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>360</y>
     <width>151</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;</string>
   </property>
   <property name="text">
    <string>乘客数</string>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumTC1lbl">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumMP1lbl">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumM2lbl">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="PassagerNumM1lbl">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>360</y>
     <width>90</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>TrainWeightTC1lbl</zorder>
  <zorder>TrainWeightMP1lbl</zorder>
  <zorder>TrainWeightM1lbl</zorder>
  <zorder>TrainWeightM2lbl</zorder>
  <zorder>TrainWeightMP2lbl</zorder>
  <zorder>TrainWeightTC2lbl</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>EmptyTrainWeightM1lbl</zorder>
  <zorder>EmptyTrainWeightM2lbl</zorder>
  <zorder>EmptyTrainWeightTC2lbl</zorder>
  <zorder>EmptyTrainWeightTC1lbl</zorder>
  <zorder>EmptyTrainWeightMP2lbl</zorder>
  <zorder>EmptyTrainWeightMP1lbl</zorder>
  <zorder>label_3</zorder>
  <zorder>PkgWeightM1lbl</zorder>
  <zorder>PkgWeightM2lbl</zorder>
  <zorder>PkgWeightTC2lbl</zorder>
  <zorder>PkgWeightTC1lbl</zorder>
  <zorder>PkgWeightMP2lbl</zorder>
  <zorder>PkgWeightMP1lbl</zorder>
  <zorder>label_4</zorder>
  <zorder>PperWeightTC2lbl</zorder>
  <zorder>PperWeightMP1lbl</zorder>
  <zorder>label_5</zorder>
  <zorder>PperWeightM1lbl</zorder>
  <zorder>PperWeightMP2lbl</zorder>
  <zorder>PperWeightM2lbl</zorder>
  <zorder>PperWeightTC1lbl</zorder>
  <zorder>PassagerNumMP2lbl</zorder>
  <zorder>PassagerNumTC2lbl</zorder>
  <zorder>label_6</zorder>
  <zorder>PassagerNumTC1lbl</zorder>
  <zorder>PassagerNumMP1lbl</zorder>
  <zorder>PassagerNumM2lbl</zorder>
  <zorder>PassagerNumM1lbl</zorder>
  <zorder>returnBtn</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
